mlflow/store/model_registry/sqlalchemy_store.py

```diff
@@ -631,6 +635,18 @@ def next_version(sql_registered_model):
         _validate_model_name(name)
         for tag in tags or []:
             _validate_model_version_tag(tag.key, tag.value)
+        storage_location = source
+        if urllib.parse.urlparse(source).scheme == "models":
+            (src_model_name, src_model_version, _, _) = _parse_model_uri(source)
```

Comment:
Does `_parse_model_uri` return a named tuple?

Reply:
If so, accessing fields by name might be safer than unpacking.

Reply:
Makes sense! I've changed the access method.